### ISSUE TITLE
attributesWhitelist does not work since version 2.45.1

### ISSUE THREAD

**Issue by abvas** [JiHong88/suneditor#1485]
**Describe the bug**
Since version 2.45.1 attributesWhitelist stopped working

I use this HTML:
```
<div style="text-align: center">
<p>This text is centered.</p>
<p>This text is centered.</p>
</div>
```

```
SUNEDITOR.create((document.getElementById('sample')),{
"width": "100%",
"height": "auto",
"attributesWhitelist": {all: "style"},
});
```

But this does not work and removes the "div" tag along with the "style" attribute, and the text is of course not centered.

Only paragraphs remain:
```
<p>This text is centered.</p>
<p>This text is centered.</p>
```

**To Reproduce**
1. Set `"attributesWhitelist": {all: "style"}`
2. Try loading HTML
```
<div style="text-align: center">
<p>This text is centered.</p>
<p>This text is centered.</p>
</div>
```

**Expected behavior**
The div with the "style" attribute should not be removed and the text should be centered.

**Codepan example**
https://codepen.io/abvas/pen/xbKKWZv?editors=1010

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser chrome,

**Additional context**
In v2.44.12 this all worked correctly.
Starting with v2.45.1 it no longer works (including in version 2.47.0)

**Comment by JiHong88** [comment]
Format line cannot contain another line.
It must be contained within a range element.
```
<div class="__se__format__range_xxx" style="text-align: center">
<p>This text is centered.</p>
<p>This text is centered.</p>
</div>

```

**Comment by abvas** [comment]
Still, this is not quite the correct behavior.
It turns out that the user cannot insert, for example, such HTML:
```
<div style="text-align: center">
<p>This text is centered.</p>
<p>This text is centered.</p>
</div>
```
the user uses external HTML and it cannot have the class "`__se__`"
it will lose formatting.

Please consider this issue again. This worked correctly before version 2.45.1

Thanks.

**Comment by davidkonrad** [comment]
Agree. I have (yet again) rolled back to v2.41.3. All versions after that is a mess. You end up using more time fixing errors (or mysterious deletions) in the generated HTML than you are using on actual writing. 

If you get the attributesWhitelist to work then it will begin to automatically delete headers, and when you fix that you cannot add attributes, if you fix that then suddenly anything is wrapped into endless spans with __custom_tag etc, after cleaning up titles are suddenly automatically removed and so on.  

The older versions had some flaws, you could live with that, but the newer versions are a mess. I have used hundreds of hours just to fix users writings / text because something odd have happen. I am a single individual who are running a free service.

Obviously the problems occur around all the attribute and tag white and black -listing. It is a great idea but we really do not **need** parential guiding with HTML, classes or attributes. And when it not work, or one rule somehow breaks other rules, why not leave it completely out, or have it as opt-in? It is nice to have, not need to have, and certainly not if it not works as expected. And why replace <q> tags with &quot; just as another example??

Well, sorry for the harsh words 🙂 Was just frustrated, have yet again used 4-5 hours to correct a text, and then finally ended up rolling back (again). Merry Christmas and a Happy New Year!! 🌲🎁🎀

**Comment by davidkonrad** [comment]
Oh, by the way, why all those `<span style="color: rgb(34, 34, 34)">` and so on everywhere?? Why change perfectly good HTML to a mess? SunEditor was so good and worked perfectly until ...Something.

**Comment by abvas** [comment]
@JiHong88 [options.strictHTMLValidation](https://github.com/JiHong88/suneditor/commit/c5b22ff25b04103574c4a307e061cec2cc2e5f58)
Sorry, but it's not entirely clear how this should work?
For example, in this example:
https://codepen.io/abvas/pen/xbKKWZv?editors=1010

**Comment by JiHong88** [comment]
Fixed in version [2.47.2](https://github.com/JiHong88/suneditor/releases/tag/2.47.2) 
The default value of the "strictHTMLValidation" option has 
Use like this
`strictHTMLValidation: false`

**Comment by davidkonrad** [comment]
For me all my problems was solved when I added strictMode: false. You should advocate for this attribute more. 

Have made a emojis plugin -> https://github.com/davidkonrad/suneditor-emojis
